jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -588,7 +589,7 @@ static JavaContainerBuilder getJavaContainerBuilderWithBaseImage(
                 "mainClass, extraClasspath, jvmFlags, and expandClasspathDependencies "
                     + "are ignored for WAR projects"));
       }
-      return null;
+      return Arrays.asList("java", "-jar", "/usr/local/jetty/start.jar");
```

Comment:
FTR: it was wrong to return this when a custom base image is specified. The issue is fixed later in #3185.